src/main/java/org/folio/service/orders/CompositePoLineValidationService.java

```diff
@@ -76,6 +76,66 @@ private List<Error> validatePoLineFormats(CompositePoLine compPOL) {
     return Collections.emptyList();
   }
 
+  public List<Error> validatePoLineMaterial(CompositePoLine compPOL) {
+
+    CompositePoLine.OrderFormat orderFormat = compPOL.getOrderFormat();
+
+    if (orderFormat == P_E_MIX) {
+      return P_E_MixCheck(compPOL);
+    } else if (orderFormat == ELECTRONIC_RESOURCE) {
+      return electronicResourceCheck(compPOL);
+    } else if (orderFormat == CompositePoLine.OrderFormat.PHYSICAL_RESOURCE) {
+      return physicalResourceCheck(compPOL);
+    } else if (orderFormat == CompositePoLine.OrderFormat.OTHER) {
+      return otherResourceCheck(compPOL);
+    }
+
+    return Collections.emptyList();
+  }
+
+  //to check if the P_E_Mix missing either physical or electronic resources
+  private List<Error> P_E_MixCheck (CompositePoLine compPOL ){
```

Comment:
java method name should start from lower case, also if we have good method name - the comment will be not necessary, propose to rename to checkPEMixFormat, also formatting should be improved
checkPEMixFormat(CompositePoLine compPOL ) {